scanner/src/lib/dataproducts/Battery_CarbonFootprint_v0.1.svelte

```diff
@@ -0,0 +1,288 @@
+  import Article from "$components/Article/index.svelte"
+  import SectionHeader from "$components/SectionHeader/index.svelte"
+  import EmissionsPieChart from "$assets/battery-emissions-chart.svg?raw"
+  import FootprintPieChart from "$assets/battery-carbon-footprint-chart.svg?raw"
+  import { inview } from "svelte-inview"
+
+  type CarbonFootprint = {
+    preProductionFootprint: number
+    mainProductionFootprint: number
+    referenceMaterial: string
+  }
+
+  export let status: number
+  export let id: string
+  export let data: {
+    batteryModel: string
+    conformityDeclaration: string
+    manufacturerInformation: {
+      name: string
+      streetName: string
+      postalCode: string
+      city: string
+      country: string
+      website: string
+      email: string
+    }
+    manufacturingLocation: {
+      city: string
+      country: string
+    }
+    carbonFootprint: CarbonFootprint
+  }
+
+  const minexpoDemo = id === "minexpo-demo-1"
+  let animateFirstChart = false
+  let animateSecondChart = false
+  let animationDelay = 500
+</script>
+
+<Article>
+  <DataRow label="Battery model" value={data.batteryModel} />
+  <DataRow label="Conformity declaration" column link value={data.conformityDeclaration} />
+  <Divider />
+  {#if !minexpoDemo}
+    <SectionHeader title="Manufacturer information">
+      The details of the battery manufacturer
+    </SectionHeader>
+    <DataRow label="Name" value={data.manufacturerInformation?.name} />
+    <DataRow label="Street name" value={data.manufacturerInformation?.streetName} />
+    <DataRow label="Postal code" value={data.manufacturerInformation?.postalCode} />
+    <DataRow label="City" value={data.manufacturerInformation?.city} />
+    <DataRow label="Country" value={countryListAlpha3[data.manufacturerInformation?.country]} />
+    <DataRow label="Website" link column value={data.manufacturerInformation?.website} />
+    <DataRow label="Email" column value={data.manufacturerInformation?.email} />
+    <Divider />
+    <SectionHeader title="Manufacturing location">
+      The details of the location of the battery manufacturing plant
+    </SectionHeader>
+    <DataRow label="City" value={data.manufacturingLocation?.city} />
+    <DataRow label="Country" value={countryListAlpha3[data.manufacturingLocation?.country]} />
+    <Divider />
+    <SectionHeader title="Carbon footprint">
+      The details of the carbon footprint for the battery production phases
+    </SectionHeader>
+    <DataRow
+      label="Pre-production footprint"
+      value={formatNumber(data.carbonFootprint?.preProductionFootprint, "kg CO2e / kWh")}
+    />
+    <DataRow
+      label="Main production footprint"
+      value={formatNumber(data.carbonFootprint?.mainProductionFootprint, "kg CO2e / kWh")}
+    />
+    <DataRow
+      label="Reference material"
+      column
+      link
+      value={data.carbonFootprint?.referenceMaterial}
+    />
+  {:else}
```

Comment:
It's not great that this is now just an `else` with no identifier for what it is. I would prefer for this to be an `if minexpoDemo` and the default case to be the `else`.